### DISCUSSION TITLE
【Puzzle 0x01】Sheet Music

### DISCUSSION THREAD

**Discussion by leeduckgo** [WeLightProject/tai-shang-world-generator#76] [Puzzles] [answered]
**State:** `Finished`

**Link:** 

> https://welightproject.github.io/tai_shang_world_generator/

**operation:**

- **Move:** ↑↓←→

- **Interact:** `space`

**Puzzle:**

NonceGeek buried `a sheet music🎼` at the coordinates `(0x01, 16, 2)`.

**Conditions met:**

↓After successfully solving the puzzle, please dm the `proof screenshot` to the twitter below↓

> https://twitter.com/Web3dAppCamp

**award:**

The first `5` successful puzzle solvers will receive a `Web3DevNFT` with a `puzzler badge (if you already have a Web3DevNFT, an additional badge will be added).

> Note: Due to the internal testing stage, there may be server connection problems. If `(0x01, 16, 2)` does not respond, please refresh the page several times and try!

## Know More: What is the Web3DevCamp Puzzle Activity?

Web3DevCamp Puzzle Event, a series of puzzle events organized by NonceGeek on TaiShangVerse -

NonceGeek will bury puzzles on different map coordinates. Players can get various rewards by entering the map and solving the puzzles (some puzzles require players to unlock a special identity!).

Among them, the map may be expressed in two forms:

- `(block_height, x, y)` , such as (24816186,2,2) refers to the NPC or treasure chest at the (2,2) coordinate on the map generated by the block data of height 24816186.

<img width="239" alt="image" src="https://user-images.githubusercontent.com/12784118/154194136-7f192d8b-0202-43d2-9f74-1416c30fc2ab.png">

- `(nft_id, x ,y)`, such as (0x01, 2, 2), refers to the NPC or treasure chest at the (2,2) coordinates on the map pointed to by the NFT numbered 1.

<img width="237" alt="image" src="https://user-images.githubusercontent.com/12784118/154193924-89dd95b3-28ba-4849-b111-e5d23fd647ae.png">

**Comment by itherunder** [comment]
1

**Comment by qiwihui** [comment]
1

**Comment by yangcancai** [comment]
1

**Reply by Zero1331** [reply]
1

**Comment by 99Kies** [comment]
1

**Comment by leeduckgo** [comment]
All `Puzzler` Badges have been sent already.
See in:
> https://gallery.noncegeek.com/

<img width="917" alt="image" src="https://user-images.githubusercontent.com/12784118/154211341-7d08ef2c-d493-45f4-aa47-004329383ce3.png">

**Comment by leeduckgo** [comment, marked as answer]
Micro-Airdrop `$10` USDC/per for all the `puzzlers`!

Tx Link:

> https://polygonscan.com/tx/0xaa4dfc591443844efe7b8a5e048b61ba6c066757de02f687e274fc1d81515bb1
